src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -127,6 +127,9 @@
   <StringProperty Name="OutputPath"
                   DisplayName="Output Path" />
 
+  <StringProperty Name="OutDir"
+                  DisplayName="OutDir Path" />
```

Comment:
Does this display anywhere? If so, I think "Output Directory" might be a better name, since "Dir" and "Path" mean the same thing (to me at least).

Also, can you move this up so the properties in the file stay in alphabetical order.

Reply:
"Output Directory" sound good.

Reply:
This doesn't display anywhere and can be removed.

Reply:
...and add `Visible="False"`.

Reply:
Removing `DisplayName` should remove the .xlf file changes too.